sections/introduction.tex

```diff
@@ -0,0 +1,31 @@
+\section{Introduction}\label{sec:introduction}
+Datasets consist of entries where each entry contains a certain amount of meta-data fields which describes the data.
```

Comment:
what is data and what is meta-data?
text i topic modeling er det data eller meta-data?